ou_dedetai/config.py

```diff
@@ -789,6 +789,14 @@ def installer_binary_dir(self) -> str:
             return self._overrides.installer_binary_dir
         return f"{self.install_dir}/data/bin"
 
+    @property
+    def logos_appdata_dir(self) -> Optional[str]:
+        """Path to the user's Logos installation under AppData"""
+        wine_user = self.wine_user
+        if not wine_user:
+            return None
+        return self.wine_prefix + f'/drive_c/users/{wine_user}/AppData/Local/Logos'
+
```

Comment:
I think this property can be used in the backup/restore code, as well. Can you take a look (`control.py`)?

Reply:
Yup line 71, good catch